### ISSUE TITLE
Demonstrate forms on pages without page builder

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#71]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [x] Developer

### Needs what

To know how to collect infomation into Xperience forms without the need of page builder.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Implemented via a15fc488
